runtime/integration-tests/src/generic/impls.rs

```diff
@@ -0,0 +1,129 @@
+/// Implements the `Runtime` trait for a runtime
+macro_rules! impl_runtime {
+	($runtime_path:ident, $kind:ident) => {
+		const _: () = {
+			use crate::generic::config::{Runtime, RuntimeKind};
+
+			impl Runtime for $runtime_path::Runtime {
+				type Api = Self;
+				type Block = $runtime_path::Block;
+				type MaxTranchesExt = $runtime_path::MaxTranches;
+				type RuntimeCallExt = $runtime_path::RuntimeCall;
+				type RuntimeEventExt = $runtime_path::RuntimeEvent;
+
+				const KIND: RuntimeKind = RuntimeKind::$kind;
+			}
+		};
+	};
+}
+
+impl_runtime!(development_runtime, Development);
+impl_runtime!(altair_runtime, Altair);
+impl_runtime!(centrifuge_runtime, Centrifuge);
+
+/// Implements fudge support for a runtime
+macro_rules! impl_fudge_support {
+	(
+        $fudge_companion_type:ident,
+        $relay_path:ident,
+        $parachain_path:ident,
+        $parachain_id:literal
```

Comment:
Awesome, we can easily extend this for adding other paras/siblings. Thank you!

Reply:
That should be the idea! 🙌🏻